Task_2/Task_2.1.2/UI/User.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Task_2._1._2.Entity;
+
+namespace Task_2._1._2.UI
+{
+    public class User
+    {
+        public string Name { get; private set; }
+
+        public List<Figure> Figures;
```

Comment:
```suggestion
        public List<Figure> Figures { get; private set; }
```
Just so someone can't replace a reference to another list